README.md

```diff
@@ -55,3 +55,7 @@ Read the DEVELOPER NOTES found in the code to understand the application
 and where you will need to make adjustments/changes as well as some 
 suggestions for best practices.
 
+Debugging server component
+--------------------------
+debugpy remote debugging enabled on port 5678 for server in docker compose, developer can attach to server from IDE e.g. vscode.
+ 
```

Comment:
The corresponding `ErrorResponses.md` from the NodeJS/React PR should be added and referenced in this readme file.